_posts/2025-08-26-post-mortem-of-find-and-save-rings-bug.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Post-Mortem of find_and_save_rings() bug"
+summary: Patched in official wallet, update highly recommended
+tags:  [core, crypto, dev, community]
+author: Jeffro256
+---
+
+{% t global.lang_tag %}
+
+## Quick Facts
+
+- Severity: HIGH if using untrusted remote daemon, MEDIUM otherwise
+- Affected versions: GUI/CLI wallet versions v0.12.0.0 to v0.18.4.1
+- Impact: Sends TXIDs of outgoing transactions to daemon after first time loading wallet from file, reducing sender anonymity
+- Fix: Update wallet to [v0.18.4.2]({{ site.baseurl_root }}/2025/08/26/monero-GUI-0.18.4.2-released.html)
```

Comment:
Should also incl CLI, especially since the reproduction steps are detailing cli usage

Reply:
Linked both